packages/dds/merge-tree/src/client.ts

```diff
@@ -704,6 +704,54 @@ export class Client {
         return segmentPosition;
     }
 
+    /**
+     * Rebases a (local) position from the perspective `{ seq: seqNumberFrom, localSeq }` to the perspective
+     * of the current sequence number. This is desirable when rebasing operations for reconnection.
+     */
+    public rebasePosition(
```

Comment:
need to add a couple unit tests for this to merge-tree package.

Reply:
also, I'm leaning toward checking in this [more inefficient] implementation first, and revisiting later to make better use of the tree structure. Assuming this approach is OK since up til' now we've been using similarly inefficient walks of the merge tree on reconnects not involving interval collections.

Reply:
added some in their own file.